### PR TITLE
Improves #1456 - Increase task launch delay only once

### PR DESCRIPTION
**Pull request by kolloch**
for unhealthy tasks, even if they do not respond to a kill
immediately.

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala [hunk 1]
```diff
@@ -26,6 +26,7 @@ class HealthCheckActor(
 
   var nextScheduledCheck: Option[Cancellable] = None
   var taskHealth = Map[String, Health]()
+  var killedTasks = Set[String]()
```
Comment (drexin) [inline src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala]: Will this ever be cleared?

Comment (drexin) [inline src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala]: Oh, this is never used.

Comment (kolloch) [inline src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala]: forgot that. nice catch. I will remove it.

**Comment by kolloch** [comment]
updated the PR.

**Comment by drexin** [comment]
:+1:

**Pull request merged**